packages/custom-elements-manifest-tools/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "@webcomponents/custom-elements-manifest-tools",
+  "version": "0.0.0",
+  "description": "Utilities for working with Custom Elements Manifest files",
+  "author": "Google LLC",
+  "license": "APACHE-2.0",
+  "type": "module",
+  "main": "index.js",
+  "scripts": {
+    "build": "wireit",
+    "test": "wireit"
+  },
+  "wireit": {
+    "build": {
+      "command": "../../node_modules/.bin/tsc --pretty",
```

```suggestion
      "command": "tsc --pretty",
```

Unless there is some reason it needs to be fully qualified?

---

It's only added as a dependency to the top level to make sure there's a single version. I'll see if the command works this way.